Cubical/Foundations/Logic.agda

```diff
@@ -273,3 +277,96 @@ Decₚ P = Dec [ P ] , isPropDec (snd P)
 ⊓-∀-distrib P Q =
   ⇒∶ (λ {(p , q) a → p a , q a})
   ⇐∶ λ pq → (fst ∘ pq ) , (snd ∘ pq)
+
+--------------------------------------------------------------------------------
+-- Introduce the "powerset" of a type in the style of Escardó's lecture notes:
+-- https://www.cs.bham.ac.uk/~mhe/HoTT-UF-in-Agda-Lecture-Notes/HoTT-UF-Agda.html#propositionalextensionality
+
+ℙ : ∀ {ℓ} → Type ℓ → Type (ℓ-suc ℓ)
+ℙ {ℓ} X = X → hProp ℓ
+
+_∈_ : {X : Type ℓ} → X → ℙ X → Type ℓ
```

Comment:
This makes definitions look so nice! Thanks @martinescardo for coming up with it